protocol/messenger_share_urls.go

```diff
@@ -45,11 +45,21 @@ type ContactURLData struct {
 	PublicKey   string `json:"publicKey"`
 }
 
+type TransactionURLData struct {
+	TxType  int    `json:"txType"`
```

Comment:
I think `tx` is redundant here, it's all about the transaction?
```suggestion
	type  int    `json:"type"`
```

Reply:
I 100% agree with you. The reason why I set it to txType to prevent any issues of `type` keyword that might happen on any layer.
While it might be redundant, I wanted to clearly point it is about transaction not url type or something like that.